backend/src/main/java/com/rota/database/DbHandler.java

```diff
@@ -0,0 +1,254 @@
+package com.rota.database;
+
+import com.rota.database.orm.DbColumn;
+import com.rota.database.orm.engagement.Engagement;
+import com.rota.database.orm.engagement.EngagementDbColumn;
+import com.rota.database.orm.engagement.EngagementMapper;
+import com.rota.database.orm.prefdates.PreferredDates;
+import com.rota.database.orm.prefdates.PreferredDatesDbColumn;
+import com.rota.database.orm.prefdates.PreferredDatesMapper;
+import com.rota.database.orm.staff.Staff;
+import com.rota.database.orm.staff.StaffDbColumn;
+import com.rota.database.orm.staff.StaffMapper;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.util.Pair;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DbHandler {
+  @Autowired
+  JdbcTemplate jdbcTemplate;
+
+  /**
+   * Erases all database entries.
+   */
+  public void wipeDb() {
+    jdbcTemplate.update(
+        "DELETE FROM STAFF WHERE TRUE;"
+            + "DELETE FROM ENGAGEMENT WHERE TRUE;"
+            + "DELETE FROM PREFERRED_DATES WHERE TRUE;"
```

Comment:
I believe this is faster `DELETE * FROM table_name;`  
And it also feels like better practice then checking if true is true